bokit/workflows/prepare_messages_for_translate.py

```
@@ -0,0 +1,20 @@
+
+    Args:
+        strings (list): The list of strings to prepare.
+        prefix (str): The prefix to prepend to each string.
+
+    Returns:
+        list: The formatted messages.
+    '''
+
+    out = []
+
+    for i, line in enumerate(strings):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)